getting-started/migrating-data.md

```diff
@@ -219,6 +205,21 @@ the number of available CPU cores on the machine.
 Above that, the workers tend to compete with each other for
 resources and reduce the performance improvements.
 
+#### Using PostgreSQL's `COPY`
+
+Although we recommend our [open sourced Go program][parallel importer]
+for better bulk insert performance, we can also use PostgreSQL's bulk insert command `COPY` to copy data 
+from the `.csv` into our new db:
+
+```bash
+psql -d new_db -c "\COPY conditions FROM old_db.csv CSV"
+```
+
+This method is straightforward and requires no extra tools, but for
+large datasets it can be impractical and time-consuming because
+`COPY` is single-threaded. For a faster method that can utilize more
+of the CPU, use the next method.
```

Comment:
`use the next method` -> `use the previous method`

Reply:
done